README.md

```diff
@@ -166,7 +166,7 @@ architectures.
 5. Invoke `.\vcpkg.exe install boost-math:x64-windows` to install the boost-math dependency.
 6. Open an "x64 Native Tools Command Prompt for VS 2019".
 7. Change directories to a location where you'd like a clone of this STL repository.
-8. Invoke `git clone https://github.com/microsoft/STL`
+8. Invoke `git clone --recurse-submodules https://github.com/microsoft/STL`

```

Comment:
I think this should have only the instructions for the submodule necessary to build since bringing in this vcpkg copy might not be relevant for folks with an ambient vcpkg available